README.md

```diff
@@ -73,7 +73,7 @@ To migrate installed packages from `2.7.4` to `2.7.5`, use `pyenv migrate`.
 
 ### License
 
-(The MIT License)
+(The MIT License)](LICENSE)
```

Comment:
I guess this markdown's link notation is corrupted.

Reply:
My bad, I think my editor erased a bracket.